2-Regression/1-Tools/translations/README.es.md

```diff
@@ -0,0 +1,206 @@
+
+## [Cuestionario previo](https://jolly-sea-0a877260f.azurestaticapps.net/quiz/9/)
+## Introducción
+
+En estas cuatro lecciones, descubrirá como crear modelos de regresión. Discutiremos para que sirven estos en breve. Pero antes de hacer cualquier coasa, asegúrese de tener las herramientas adecuadas para comenzar el proceso!
+
+En esta lección, aprenderá a:
+
+- Configurar su computadora para tares locales de machine learning.
+- Trabajar con cuadernos Jupyter.
+- Usar Scikit-learn, incluida la instalación.
+- Explorar la regressión lineal con un ejercicio práctico.
+
+## Instalaciones y configuraciones.
+
+[![Uso de Python con Visual Studio Code](https://img.youtube.com/vi/7EXd4_ttIuw/0.jpg)](https://youtu.be/7EXd4_ttIuw "Uso de Python con Visual Studio Code")
+
+> 🎥 Haga click en la imagen de arriba para ver un video: usando Python dentro de VS Code.
+
+1. **Instale Python**. Asegúrese de que [Python](https://www.python.org/downloads/) esté instalado en su computadora. Utilizará Python para muchas tareas de ciencia de datos y machine learning. La mayoría de los sistemas informáticos ya incluyen una instalación de Python. También hay disponibles [paquetes de código de Python](https://code.visualstudio.com/learn/educators/installers?WT.mc_id=academic-15963-cxa) útiles para facilitar la configuración a algunos usuarios.
+
+   Sin embargo algunos usos de Python requieren una versión del software, mientras otros requieren una versión diferente. Por esta razón, es útil trabajar dentro de un [entorno virtual](https://docs.python.org/3/library/venv.html).
+
+2. **Instale Visual Studio Code**. Asegúrese de tener Visual Studio Code instalado en su computadora. Siga estas instrucciones para [instalar Visual Studio Code](https://code.visualstudio.com/) para la instalación básica. Va a utilizar Python en Visual Studio Code en este curso, por lo que es posible que desee repasar cómo [configurar Visual Studio Code](https://docs.microsoft.com/learn/modules/python-install-vscode?WT.mc_id=academic-15963-cxa) para el desarrollo en Python.
+
+   > Siéntase cómodo con Python trabajando con esta colección de [módulos de aprendizaje](https://docs.microsoft.com/users/jenlooper-2911/collections/mp1pagggd5qrq7?WT.mc_id=academic-15963-cxa)
+
+3. **Instale Scikit-learn**, siguiendo [estas instrucciones](https://scikit-learn.org/stable/install.html). Dado que debe asegurarse de usar Python3, se recomienda que use un entorno virtual. Tenga en cuenta que si está instalando esta biblioteca en una Mac M1, hay instrucciones especiales en la página vinculada arriba.
+
+1. **Instale Jupyter Notebook**. Deberá [instalar el paquete de Jupyter](https://pypi.org/project/jupyter/). 
+
+## El entorno de creación de ML
+
+Va a utilizar **cuadernos** para desarrollar tu código Python y crear modelos de machine learning. Este tipo de archivos es una herramienta común para científicos de datos, y se pueden identificar por su sufijo o extensión `.ipynb`.
```

Comment:
Not trying to nitpick, but this is still going from formal (_Va a utilizar_) to informal (_desarrollar tu código_).

Reply:
To have the opportunity to learn from people like you, it's a blessing, please be as demanding as possible with my work, I want to learn and be successful. Thanks!

Reply:
You can contact me and share any information that helps me do a better job.